packages/perseus/src/widgets/__stories__/radio.stories.tsx

```diff
@@ -9,78 +9,78 @@ import {
 
 import type {PerseusRenderer} from "../../perseus-types";
 import type {APIOptions} from "../../types";
+import type {Meta} from "@storybook/react";
 
 type StoryArgs = {
+    // Story Option
+    question: PerseusRenderer;
     // Radio Options
     static: boolean;
     // API Options
     crossOutEnabled: boolean;
-    // Renderer Options
-    reviewMode: boolean;
-};
-
-type Story = {
-    title: string;
-    args: StoryArgs;
-};
+} & Pick<
+    React.ComponentProps<typeof RendererWithDebugUI>,
+    "reviewMode" | "showSolutions"
+>;
 
 export default {
     title: "Perseus/Widgets/Radio",
     args: {
         static: false,
         crossOutEnabled: false,
         reviewMode: false,
+        showSolutions: "none",
+        question: questionWithPassage,
```

Comment:
By adding `question` to args, I accidentally made an interactive and more grokkable version of the Perseus JSON panel!

<img width="1741" alt="Screenshot 2024-04-17 at 1 45 08 PM" src="https://github.com/Khan/perseus/assets/23404711/ecdbdb96-9ab0-45c8-86ec-3c122d215d80">